### DISCUSSION TITLE
3163 plots

### DISCUSSION THREAD

**Discussion by krasher9** [Chia-Network/chia-blockchain#6951] [General]
I have 3163 plots and sadly to see its been 40 days with no win. Hope the pool gets released :/ this is kinda tilting.

**Comment by alexwwstx** [comment]
Statistically you should have at least one hit, unless something else is not right

**Reply by krasher9** [reply]
well. i havent till now and no idea whats going on.

**Comment by IgnaciusMaximus** [comment]
Check your log file in order to see if your farmer/harvester are comunicating correctly.

on the other hand, expected time to win could be times 2 or 3.

**Reply by krasher9** [reply]
![SSSS](https://user-images.githubusercontent.com/83899298/123416217-5539ed80-d5c7-11eb-94a3-370c934dde17.jpg)

**Reply by krasher9** [reply]
![image](https://user-images.githubusercontent.com/83899298/123416543-bf529280-d5c7-11eb-923b-f63d9c3ec9a9.png)

**Comment by jcodic** [comment]
Don't worry man, it's new normal. I have 2800+ plots and 30 days without a block. Desperately waiting for pools.
